### PR TITLE
Add: Sigma Dao Screenshots

### PR DESCRIPTION
**Pull request by rmeena840**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 <img src="./src/assets/logo.png" height="200" title="Sigma Dao" />
 
+# SigmaDao Glimpes:
```
Comment (robert-zaremba) [inline README.md]:
```suggestion
## SigmaDAO glimpes:
```

**Review by robert-zaremba — COMMENTED** [review]
1. Let's move screenshots down, just above the _Requirements_ section
2. Add some description to every screenshot

**Comment by rmeena840** [comment]
@robert-zaremba Suggested changes done. Ready for review again.

**Comment by rmeena840** [comment]
@robert-zaremba Please review the PR.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,29 @@
-<img src="./src/assets/logo.png" height="200" title="Sigma Dao" />
+<img src="./src/assets/logo.png" height="200" title="SigmaDAO" />
 
-# Sigma Dao
+# SigmaDAO
 
-Sigma Dao is a web application that let's a user create a DAO, enable them to Add Proposal, Deposit and Withdraw Tokens and also allow them to Vote.
+SigmaDAO is a web application that let's a user create a DAO, enable them to Add Proposal, Deposit and Withdraw Tokens and also allow them to Vote.
```
Comment (robert-zaremba) [inline README.md]:
```suggestion
SigmaDAO is a web application that lets anyone create a DAO, enable them to Add Proposal, Deposit and Withdraw Tokens and also allow them to Vote.
```

File: README.md [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
 
-Wallets supported by Sigma Dao are:
+Wallets supported by SigmaDAO are:
```
Comment (robert-zaremba) [inline README.md]:
```suggestion
Wallets supported by the SigmaDAO are:
```

**Review by robert-zaremba — APPROVED** [review]

**Pull request merged**